R-packages/evalcast/R/signal_listcols.R

```diff
@@ -0,0 +1,32 @@
+signal_listcols <- function(signals, forecast_date) {
```

Comment:
Please add some documentation as to what this function is supposed to be doing.

Reply:
done.